config-model/src/main/java/com/yahoo/vespa/model/container/ContainerCluster.java

```diff
@@ -187,6 +187,7 @@ public ContainerCluster(AbstractConfigProducer<?> parent, String configSubId, St
         addSimpleComponent("com.yahoo.container.jdisc.ContainerThreadFactory");
         addSimpleComponent("com.yahoo.container.handler.VipStatus");
         addSimpleComponent(com.yahoo.container.handler.ClustersStatus.class.getName());
+        addSimpleComponent("com.yahoo.container.jdisc.DisabledConnectionLogProvider");
```

Comment:
Won't you get 2 conflicting providers for application clusters then ?

Reply:
No, a provider acts as a fallback if not component implementing `ConnectionLog` is configured.